pkg/apis/componentconfig/v1alpha1/defaults.go

```diff
@@ -396,7 +393,7 @@ func SetDefaults_KubeletConfiguration(obj *KubeletConfiguration) {
 		obj.HairpinMode = PromiscuousBridge
 	}
 	if obj.EvictionHard == nil {
-		temp := "memory.available<100Mi"
+		temp := "memory.available<100Mi,nodefs.available<10%,nodefs.inodesFree<5%"
```

Comment:
actually, quick question here... why make this a percentage and not a literal value same as the old lowDiskSpace default of 256MB?